### ISSUE TITLE
.mobi version

### ISSUE THREAD

**Issue by timkofu** [jeffgerickson/algorithms-ftw#171]
Thank you for making this book free.

Would it be possible to publish a .mobi of it for eReaders?

Thanks again.

**Comment by busfault** [comment]
Yes please! I attempted with pdf and with the OCR'd one at archive.org (https://archive.org/details/Algorithms-Jeff-Erickson/). Which would need a significant amount of proofreading to get to a usable state. I'd help with this effort if need be.

Edit:
This is duplicate of  #16
